pkg/webhook/storageclass.go

```diff
@@ -165,11 +109,16 @@ func applyV1StorageClassPatch(sc *storagev1.StorageClass) *v1.AdmissionResponse
 		return reviewResponse
 	}
 
-	if _, ok := sc.Parameters["instancePrefix"]; ok {
+	tier, ok := sc.Parameters["tier"]
```

Comment:
before checking tier, we should also verify that any string other than true, True, false, False for "multishare" should be rejected.

Reply:
ack